src/mapml-viewer.js

```diff
@@ -354,6 +354,19 @@ export class MapViewer extends HTMLElement {
       totalSize += 49;
       this._fullScreenControl = M.fullscreenButton().addTo(this._map);
     }
+    if (this.getAttribute("controlslist") === "geolocation" && !this._geolocationButton && (totalSize + 49) <= mapSize){
```

Comment:
Since the geolocation control is no longer at the top left of the map, I don't believe it needs to be taken into consideration in the totalSize calculation. If I recall correctly the purpose of this calculation, it's to not add a control if it will not be visible on a very small map; since we moved this control to the bottom right, it shouldn't be considered for that purpos.